src/Swagger.ObjectModel/Utilities.cs

```diff
@@ -5,9 +5,14 @@ namespace Swagger.ObjectModel
 {
     public static class Utilities
     {
+        /// <summary>
+        /// convert a string to camelcase with special rules.
+        /// </summary>
```

Comment:
Not a fan of redundant comments usually

Reply:
All right , this comment needs to be remove !